staging/dex-k8s-authenticator/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "v1.1.0"
+description: "Authenticator for using Dex with Kubernetes"
+name: dex-k8s-authenticator
+version: 1.1.1
```

Comment:
Note: This version doesn't need to be the same version supported in the upstream repo, but it helps to identify the version used in these scripts.